### PR TITLE
Use the new toolbox version (Go version)

### PR DESCRIPTION
**Pull request by andrunko**
The new package is named podman-toolbox instead of toolbox.

https://phabricator.endlessm.com/T31300

### CODE REVIEW

**Comment by andrunko** [comment]
CI is failing because the package is still not in our repos - pushed an OBS PR to add it, once it lands the CI should be green again.

**Review by wjt — APPROVED** [review]
LGTM once the package lands.

**Comment by andrunko** [comment]
retest this please.

**Comment by andrunko** [comment]
retest this please.

**Pull request merged**
